sushichef.py

```diff
@@ -417,12 +353,8 @@ def build_english_video_topics(topic):
         print('==> Download of Videos FAILED!')
         return False
 
-    # TODO(cpauya: VideoNode constructor has no argument for language code?
+    # NOTE(cpauya: VideoNode constructor has no argument for language code?
     for i, video in enumerate(video_data):
-        # TODO(cpauya): Remove when done testing
-        # if i > 0:
-        #     break
-        # print('==> video', video)
         filepath = video.filepath
         video_node = VideoNode(
                 source_id=video.uid, 
```

Comment:
Please change the next line to:
`title=video.title.replace('(English Language)', '').strip()`


this will remove the repeated string "English Language" that appears in all the videos.